bundles/org.openhab.binding.radiothermostat/README.md

```diff
@@ -0,0 +1,211 @@
+
+## Discovery
+
+Auto-discovery is supported if the thermostat can be located on the local network using SSDP. Otherwise the thing must be manually added.
+
+## Binding Configuration
+
+The binding has no configuration options, all configuration is done at Thing level.
+
+## Thing Configuration
+
+The thing has a few configuration parameters:
+
+|    Parameter    | Description                                                                                               |
+|-----------------|-----------------------------------------------------------------------------------------------------------|
+| hostName        | The host name or IP address of the thermostat. Mandatory.                                                 |
+| refresh         | Overrides the refresh interval of the thermostat data. Optional, the default is 2 minutes.                |
+| logRefresh      | Overrides the refresh interval of the run-time logs & humidity data. Optional, the default is 10 minutes. |
+| disableLogs     | Disable retrieval of run-time logs from the thermostat. Optional, the default is 0.                       |
+| disableHumidity | Disable retrieval of humidity information from the thermostat. Optional, the default is 0.                |
+
+## Channels
+
+The thermostat information that is retrieved is available as these channels:
+
+| Channel ID             | Item Type            | Description                                                               |
+|------------------------|----------------------|---------------------------------------------------------------------------|
+| name                   | String               | The name of the thermostat                                                |
+| model                  | String               | The model number and firmware version of the thermostat                   |
+| temperature            | Number:Temperature   | The current temperature reading of the thermostat                         |
+| humidity               | Number               | The current humidity reading of the thermostat (CT80 only)                |
+| mode                   | Number               | The current operating mode of the HVAC system                             |
+| fan_mode               | Number               | The current operating mode of the fan                                     |
+| program_mode           | Number               | The program schedule that the thermostat is running (CT80 Rev B only)     |
+| set_point              | Number:Temperature   | The current temperature set point of the thermostat                       |
+| status                 | Number               | Indicates the current running status of the HVAC system                   |
+| fan_status             | Number               | Indicates the current fan status of the HVAC system                       |
+| override               | Number               | Indicates if the normal program set-point has been manually overridden    |
```

Comment:
I did this to match the display on the thermostat (using transform in the example) and because it is read-only in the api. It can't be turned off so having a switch here would be confusing IMHO.